xbmc/pvr/windows/GUIWindowPVRBase.cpp

```diff
@@ -389,7 +389,20 @@ bool CGUIWindowPVRBase::OpenChannelGroupSelectionDialog(void)
 
 bool CGUIWindowPVRBase::InitChannelGroup()
 {
-  CPVRChannelGroupPtr group(CServiceBroker::GetPVRManager().GetPlayingGroup(m_bRadio));
+  std::shared_ptr<CPVRChannelGroup> group;
+  if (m_channelGroupPath.empty())
+  {
+    group = CServiceBroker::GetPVRManager().GetPlayingGroup(m_bRadio);
+  }
+  else
+  {
+    group = CServiceBroker::GetPVRManager().ChannelGroups()->Get(m_bRadio)->GetGroupByPath(m_channelGroupPath);
+    if (group)
```

Comment:
Should this be outside the else-block?

Reply:
No, I don't want last played group as fallback. This only would hide errors - e.g. if a skinner does supply an  invalid channel group path.

Makes sense?

Reply:
`SetPlayingGroup` is only needed if `group` was not obtained using `GetPlayingGroup`. 
Error message is specific for obtaining `group` using `GetGroupByPath`, 

All fine, imo. ;-)